lib/utils/file_uploader.dart

```diff
@@ -282,6 +283,29 @@ class FileUploader {
     }
   }
 
+
+  Future<void> removeStaleFiles() async {
+    try {
+      final String dir = Configuration.instance.getTempDirectory();
```

Comment:
Would it make more sense to create a sub-dir for keeping these files and just removing that?

Reply:
I initially thought of starting with that but the current configuration file is little too much and confusing and didn't want to refactor that yet. We will fix these when we pick resumable upload and download